etc/config/c.amazon.properties

```diff
@@ -290,6 +290,16 @@ compiler.craspbian9.exe=/opt/compiler-explorer/arm/raspbian9-6.3.0/bin/arm-raspb
 compiler.craspbian9.name=Raspbian Stretch
 compiler.craspbian9.semver=6.3
 compiler.craspbian9.objdumper=/opt/compiler-explorer/arm/raspbian9-6.3.0/bin/arm-raspbian9-linux-gnueabihf-objdump
+compiler.arduinouno.exe=/opt/compiler-explorer/avr/arduino-1.8.9/hardware/tools/avr/bin/avr-gcc
+compiler.arduinouno.options=-std=gnu++11 -fpermissive -fno-exceptions -ffunction-sections -fdata-sections -fno-threadsafe-statics -mmcu=atmega328p -DF_CPU=16000000L -DARDUINO=10809 -DARDUINO_AVR_UNO -DARDUINO_ARCH_AVR -I/opt/compiler-explorer/avr/arduino-1.8.9/hardware/arduino/avr/variants/standard -I/opt/compiler-explorer/avr/arduino-1.8.9/hardware/arduino/avr/cores/arduino
+compiler.arduinouno.name=Arduino Uno
+compiler.arduinouno.semver=5.4.0
+compiler.arduinouno.objdumper=/opt/compiler-explorer/avr/arduino-1.8.9/hardware/tools/avr/bin/avr-objdump
+compiler.arduinouno.exe=/opt/compiler-explorer/avr/arduino-1.8.9/hardware/tools/avr/bin/avr-gcc
```

Comment:
as above can you please put some version info, and fix up the duplicated "uno" config id